pkgs/misc/emulators/rpcs3/update.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p gnused jq nix-prefetch-git git
+
+set -eou pipefail
+
+ROOT="$(dirname "$(readlink -f "$0")")"
+if [ ! "$(basename $ROOT)" = "rpcs3" ] || [ ! -f "$ROOT/default.nix" ]; then
```

Comment:
That is probably specific to vscodium.

> [[ and ]] are bashisms, and I want to keep them out.

Why? NixOS uses bash everywhere and there is no real advantage of not using it. Also bashisms especially new test make your life with shell much easier and prevents many common errors.

Reply:
> That is probably specific to vscodium.
> 
It works, even when run through the maintainer update script.

> > [[ and ]] are bashisms, and I want to keep them out.
> 
> Why? NixOS uses bash everywhere and there is no real advantage of not using it. Also bashisms especially new test make your life with shell much easier and prevents many common errors.

Honestly, because I find it easier to read. But whatever, this isn't a hill worth dying on. I'll update it after $dayjob.